zenoh-pico-dds/roundtrip/README.md

```diff
@@ -0,0 +1,53 @@
+# RoundTrip
+The Roundtrip example allows the measurement of roundtrip duration when sending and receiving back a single message
+
+This example is compatible with CycloneDDS' [roundtrip example](https://github.com/eclipse-cyclonedds/cyclonedds/tree/0.10.2/examples/roundtrip) (sources copied and built here for convenience).
+
+## Prerequisite:
+
+ - CycloneDDS (>=0.11 or master branch) installed and its `idlc` code generator accessible in the `${PATH}`
+ - CMake 2.14+
+ - [zenoh-bridge-dds](https://github.com/eclipse-zenoh/zenoh-plugin-dds)
+
+## How to build:
+
+```bash
+mkdir build
+cd build
+cmake ..
+make
+```
+
+## Result of build:
+
+ - **`RoundtripPing`**: CycloneDDS' ping example.
+
+ - **`RoundtripPong`**: CycloneDDS' pong example.
+
+ - **`z_ping`**: A zenoh-pico equivalent to RoundtripPing, sending a ping message and waiting for a poing message in return.  
```

Comment:
@JEnoch typo "poing"

Reply:
Thanks for spotting this @bryceschober !
Fixed in https://github.com/eclipse-zenoh/zenoh-demos/commit/ea6e8fc781bd49d532389abc18be5a42a23ccdba